src/sensors/utils.rs

```diff
@@ -225,12 +228,22 @@ impl ProcessTracker {
                             }
                         }
                         found = true;
-                    } else if self.regex_cgroup_kubernetes.is_match(&cg.pathname) {
-                        // kubernetes
-                        description.insert(
-                            String::from("container_scheduler"),
-                            String::from("kubernetes"),
-                        );
+                    } else {
+                        // containerd
+                        if self.regex_cgroup_containerd.is_match(&cg.pathname) {
+                            description.insert(
+                                String::from("container_runtime"),
+                                String::from("containerd"),
+                            );
+                        // kubernetes using docker
+                        } else if self.regex_cgroup_kubernetes.is_match(&cg.pathname) {
+                            description
+                                .insert(String::from("container_runtime"), String::from("docker"));
+                        } else {
+                            // cgroup not related to a container technology
+                            continue;
```

Comment:
I'd like to get feedback on this part as I wasn't sure on it. 

We need to check if there is a matching pod when the regex matches containerd or kubernetes. If neither regex matches should we keep checking the cgroup paths?